docs/gettsim_objects/input_variables.rst

```diff
@@ -90,3 +88,13 @@ household.
 +-------------------------+----------------------------------------------+-------------+
 | _`entgeltpunkte`        | Earning points for pension claim             | FloatSeries |
 +-------------------------+----------------------------------------------+-------------+
+| _`g_r_bewertungszeiten` | Months with > 30% of average income          | IntSeries   |
++-------------------------+----------------------------------------------+-------------+
+| _`entgeltp_grundr`      | Entgeltpunkte in Grundrentenbewertungszeiten | FloatSeries |
++-------------------------+----------------------------------------------+-------------+
+| _`grundrentenzeiten`    | Number of months considered for Grundrente   | IntSeries   |
++-------------------------+----------------------------------------------+-------------+
+| _`priv_rente_m`         | Amount of monthly private pension            | FloatSeries |
++-------------------------+----------------------------------------------+-------------+
+| _`schwerbe_ausweis_g`   | Dummy: Schwerbehindertenausweis Merkzeichen G| BoolSeries  |
```

Comment:
I find this abbreviation rather unnatural, in the middle of a syllable. How about `schwerbeh_ausweis_g` ? If too long, `schwerbeh_ausw_g` or `schwerb_ausweis_g`?